File: WarehouseManagementWeb.Application/Dto/Input/Client/ChangeStatusClientInput.cs
using WarehouseManagementWeb.Domain.Enums;

namespace WarehouseManagementWeb.Application.Dto.Input.Client
{
    /// <summary>
    /// Класс входной модели обновления статуса клиента.
    /// </summary>
    public class ChangeStatusClientInput
    {
        /// Id клиента.
        /// </summary>
        public int ClientId { get; set; }

        /// <summary>
        /// Статус клиента.
        /// </summary>
        public DirectoryStatusEnum ClientStatusEnum { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/directory/client/models/input/change-status-client-input.ts
import { DirectoryStatusEnum } from '../../../enums/directory-status-enum';

/**
 * Класс входной модели обновления статуса клиента.
 */
export class ChangeStatusClientInput {
  /**
   *  Id клиента.
   */
  clientId: number = 0;

  /**
   * Статус клиента.
   */
  clientStatusEnum: DirectoryStatusEnum = 0;
}
